api/v1beta1/hyperconverged_types.go

```diff
@@ -25,6 +25,18 @@ const (
 	HyperConvergedUninstallStrategyBlockUninstallIfWorkloadsExist HyperConvergedUninstallStrategy = "BlockUninstallIfWorkloadsExist"
 )
 
+// TODO: Not sure if it needs the openapi-gen this should be a corev1.ConfigMap
+
+type Policy struct {
+	Name string `json:"name,omitempty"`
```

Comment:
What do you think about using a pointer instead of a string?
```suggestion
	Name *string `json:"name,omitempty"`
```

Reply:
I have changed a bit the approach on this.